packages/types/src/worktemplates.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {RequireOnlyOne} from './utilities';
+
+export type WorkTemplatesState = {
+    categories: Category[];
+    templatesInCategory: Record<string, WorkTemplate[]>;
+}
+
+export interface WorkTemplate {
+    id: string;
+    category: Category;
+    useCase: string;
+    description: Description;
+    illustration: string;
+    visibility: Visibility;
+    content: ValidContent[];
+}
+
+export const categories = ['product', 'devops', 'company_wide', 'leadership', 'design'];
+export interface Category {
+    id: typeof categories[number];
+    name: string;
+}
+
+export interface Channel {
+    id: string;
+    name: string;
+    illustration: string;
+}
+export interface Board {
+    id: string;
+    name: string;
+    illustration: string;
+}
+export interface Playbook {
```

Comment:
these interfaces have equal body (Channel, Board, Playbook), you could declare one boilerplate and then use type (under the hood interfaces and types are the same)

Reply:
This is a temporary situation, we'll have more fields in each of them once the API is plugged in.

Reply:
R

Reply:
Let's leave a TODO comment on that so that there is more context for others that stumble into this later